io/net/virt-net/network_virtualization.py

```diff
@@ -547,10 +510,9 @@ def device_add_remove(self, slot, mac, operation):
             cmd = 'chhwres -m %s --id %s -r virtualio --rsubtype vnic \
                    -o r -s %s'\
                    % (self.server, self.lpar_id, slot)
-        try:
-            self.run_command(self.con_hmc, cmd)
-        except CommandFailed as cmdfail:
-            self.log.debug(str(cmdfail))
+        output = self.session_hmc.cmd(cmd)
+        if not output.exit_status == 0:
+            self.log.debug(output.stderr)
```

Comment:
try..except is removed, is that deliberate ?

Reply:
it's because in login() ,run_command() for handling exception we are using "class CommandFailed(Exception):" this class. but now it will take care in ssh module.

Reply:
okay